checker/bin-devel/javac

```diff
@@ -35,7 +35,7 @@ annoToolsDir="${cfDir}"/../annotation-tools
 # Put afu jar files last, as they might contain out-of-date CF files.
 # Put "checker" after the other sub-projects, as "ant bindist" puts
 # other projects into the checker/build directory.
-buildDirs="${cfDir}"/dataflow/build:"${cfDir}"/javacutil/build:"${cfDir}"/stubparser/build:"${cfDir}"/framework/build:"${cfDir}"/checker/build:"${annoToolsDir}"/scene-lib/bin:"${annoToolsDir}"/annotation-file-utilities/annotation-file-utilities.jar
+buildDirs="${cfDir}"/dataflow/build:"${cfDir}"/javacutil/build:"${cfDir}"/framework/build:"${cfDir}"/checker/build:"${cfDir}"/checker/lib/javaparser-core.jar:"${annoToolsDir}"/scene-lib/bin:"${annoToolsDir}"/annotation-file-utilities/annotation-file-utilities.jar
```

Comment:
@panacekcz, @mernst it's true, the path should be changed because the lib is in the framework/lib/. I will change it and rename to the stubparser-core.jar.